pages/index.jsx

```diff
@@ -0,0 +1,54 @@
+import Head from "next/head";
+import Image from "next/Image";
+import styles from "../styles/Home.module.scss";
+import profilePic from "../public/pictures/meo.jpg";
+import profilePic2 from "../public/pictures/background.jpg"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faFire } from "@fortawesome/free-solid-svg-icons";
+import { faStar } from "@fortawesome/free-regular-svg-icons";
+import { COLORS } from "../public/constants.jsx"
+
+export default function Home() {
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>VƯỜN HOA SỨ</title>
+        <meta
+          name="description"
+          content="Dự án vũ trụ truyện thuộc MEO Studio"
+        />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main className={styles.main}>
+        <div className={styles.gridContainer}>
```

Comment:
I think you misunderstand what I mean. The whole hero section (including the pictures and the title) should be under one div that is also a grid